### ISSUE TITLE
Can this work with real data?

### ISSUE THREAD

**Issue by vzhilov** [tonioluna/fake_sky_chart_generator#1]
Thank you for sharing you code, it happen this is exactly what I need!  But I’m trying to make it work with the real space research data, like astropy, showing a target star with some of its neighbors . Would you be available to make such a feature?

**Comment by tonioluna** [comment]
I don't think that would be easy to do. The code I have does not know anything about a real coordinate system. However, I have used this one in the past to generate charts out of actual data: https://github.com/codebox/star-charts

**Comment by vzhilov** [comment]
Thank you for your fast response.

It seems like if we change `get_random_stars` function, partially the lines:

       w = random.random()*max_w
       h = random.random()*max_h

to the real coordinates gotten by `astropy`:

```
from astropy.coordinates import SkyCoord 
from astropy import units as u
c = SkyCoord(ra=ra1*u.degree, dec=dec1*u.degree)  
catalog = SkyCoord(ra=ra2*u.degree, dec=dec2*u.degree)
idxc, idxcatalog, d2d, d3d = catalog.search_around_sky(c, 1*u.deg)
```

we could get the real sky chart with some stars around. 

Am I thinking in the right way?

The other script of `codebox` you have mentioned looks good but I couldn't find if it is possible to change the graphics to display it in a style like yours.

**Comment by tonioluna** [comment]
I'm not familiar with Astropy. Does it provide standard x, y coords? Is
some projection requiered?

I sounds like an interesting idea. I suppose github supports branches on
public projects. Feel free to branch, play around, and once it's working it
can be merged back to master.
